permission denied when running `make build`

Hey, Guys

I followed this [building doc](https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/devguide.md#2-clone-fork-to-local-storage) to build the images but got the below errors, could you help to check this issue? Or something else I missed?

```command
# make build
docker run --rm -v /home/goproject/src/github.com/kubernetes-incubator/service-catalog:/go/src/github.com/kubernetes-incubator/service-catalog -v /home/goproject/src/github.com/kubernetes-incubator/service-catalog/.pkg:/go/pkg scbuildimage go build -o bin/defaulter-gen github.com/kubernetes-incubator/service-catalog/vendor/k8s.io/code-generator/cmd/defaulter-gen
can't load package: package github.com/kubernetes-incubator/service-catalog/vendor/k8s.io/code-generator/cmd/defaulter-gen: open /go/src/github.com/kubernetes-incubator/service-catalog/vendor/k8s.io/code-generator/cmd/defaulter-gen: permission denied
Makefile:138: recipe for target 'bin/defaulter-gen' failed
make: *** [bin/defaulter-gen] Error 1
```
Relevnt info:
```
# go version
go version go1.9.2 linux/amd64
[# echo $GOPATH
/home/goproject
# echo $GOROOT
/usr/local/go
[root@dhcp-140-165 service-catalog]# pwd
/home/goproject/src/github.com/kubernetes-incubator/service-catalog
[root@dhcp-140-165 service-catalog]# git branch
* master
[root@dhcp-140-165 service-catalog]# uname -a
Linux dhcp-140-165.nay.redhat.com 4.8.6-300.fc25.x86_64 #1 SMP Tue Nov 1 12:36:38 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
[root@dhcp-140-165 service-catalog]# docker version
Client:
 Version:         1.12.6
 API version:     1.24
 Package version: docker-common-1.12.6-7.gitae7d637.fc25.x86_64
 Go version:      go1.7.6
 Git commit:      ae7d637/1.12.6
 Built:           Tue Jul 18 16:18:12 2017
 OS/Arch:         linux/amd64

Server:
 Version:         1.12.6
 API version:     1.24
 Package version: docker-common-1.12.6-7.gitae7d637.fc25.x86_64
 Go version:      go1.7.6
 Git commit:      ae7d637/1.12.6
 Built:           Tue Jul 18 16:18:12 2017
 OS/Arch:         linux/amd64

```

--- a/Makefile
+++ b/Makefile
@@ -97,7 +97,7 @@ ifdef NO_DOCKER
 	scBuildImageTarget =
 else
 	# Mount .pkg as pkg so that we save our cached "go build" output files
-	DOCKER_CMD = docker run --rm -v $(PWD):/go/src/$(SC_PKG) \
+	DOCKER_CMD = docker run --security-opt label:disable --rm -v $(PWD):/go/src/$(SC_PKG) \
 	  -v $(PWD)/.pkg:/go/pkg scbuildimage
 	scBuildImageTarget = .scBuildImage
 endif

